diff --git a/node_modules/@inkjs/ui/build/components/multi-select/use-multi-select.js b/node_modules/@inkjs/ui/build/components/multi-select/use-multi-select.js
index e5461cd..a86a08f 100644
--- a/node_modules/@inkjs/ui/build/components/multi-select/use-multi-select.js
+++ b/node_modules/@inkjs/ui/build/components/multi-select/use-multi-select.js
@@ -7,7 +7,7 @@ export const useMultiSelect = ({ isDisabled = false, state, }) => {
         if (key.upArrow) {
             state.focusPreviousOption();
         }
-        if (input === ' ') {
+        if (key.tab) {
             state.toggleFocusedOption();
         }
         if (key.return) {
